releasenotes/notes/add-django-channels-support-44aefaa21961adff.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Django channels currently fails on startup when ddtrace is enabled.
+    This change updates our django view instrumentation to support django-channels v3.0.
```

Is this only an issue for `django-channels>=3.0` ?

We should also word this as stating that something is fixed, not that something is currently a problem.

```suggestion
    Fix application crash on startup when using ``django-channels>=3.0``.
```